docs/code/local-setup.md

```diff
@@ -117,29 +118,32 @@ you need to create access keys and configure your local setup to use them:
 
 ## Configure dbt
 
-The connection information for our data warehouses will,
+dbt core was installed when you created your infra environment and ran the poetry command. The connection information for our data warehouses will,
 in general, live outside of this repository.
-This is because connection information is both user-specific usually sensitive,
-so should not be checked into version control.
+This is because connection information is both user-specific and usually sensitive,
+so it should not be checked into version control.
+
 In order to run this project locally, you will need to provide this information
-in a YAML file located (by default) in `~/.dbt/profiles.yml`.
+in a YAML file. Run the following command to create the necessary folder and file.
+
+```bash
+mkdir ~/.dbt && touch ~/.dbt/profiles.yml
```

Comment:
👍 Nice addition. Maybe we should call out that this is only expected to work on posix-y systems, and Windows users will have a different command. In general, I have found this to be a very challenging step for some users who are unaccustomed to creating folders and files in specific places.

Reply:
Thank you for that note!